public/index.html

```diff
@@ -10,6 +10,9 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```

Comment:
@Trispa on ajoute pas bootstrap ici dans le `public/index.html` et l'obtenir directement d'un cdn externe. C'est une dépendence à un cdn externe que l'on peut éviter.
Et en plus que bootstrap est déjà importé dans index.js, donc pas besoin de ça

Reply:
see :    https://react-bootstrap.github.io/getting-started.html

"Because React-Bootstrap doesn't depend on a very precise version of Bootstrap, we don't ship with any included css. However, some stylesheet is required to use these components. How and which bootstrap styles you include is up to you, but the simplest way is to include the latest styles from the CDN."